howto/consume-topic/README.md

```diff
@@ -76,6 +76,27 @@ app.post('/orders', (req, res) => {
 app.listen(port, () => console.log(`consumer app listening on port ${port}!`))
 ```
 
+#### Subscribing on Kubernetes
+
+In Kubernetes, save the CRD to a file and apply it to the cluster:
+
+```
+kubectl apply -f subscription.yaml
+```
+
+#### Subscribing in Self Hosted
+
+When running Dapr in Self-Hosted, either locally or on a VM, put the CRD in your `./components` directory.
```

Comment:
nit Self-hosted

Reply:
Changed

Reply:
This is probably org-level nit but given Dapr components are the same format regardless of the deployment target (self-hosted or k8s) may be clearer to skip the CRD nomenclature in self-hosted deployments and talk about deployment of components to k8s instead of "applying CRD".